### PR TITLE
 Naming update for Stacks rebrand #841 

### PR DESCRIPTION
**Pull request by friedger**
## Description

This PR replaces the use of the name `Blockstack` with `Stacks` where it makes sense.

Some naming changes require an additional noun like `Blockstack` in the context of authentication becomes `Stacks Auth`.

For details refer to issue #841 

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [x] API reference/documentation update
- [ ] Other

## Does this introduce a breaking change?
No

## Are documentation updates required?
No

## Testing information
Nothing special

## Checklist
- [x] Code is commented where needed
- [x] Unit test coverage for new or modified code paths
- [x] `npm run test` passes
- [x] Changelog is updated
- [x] Tag 1 of @agraebe @GinaAbrams

### CODE REVIEW

File: src/pages/data-storage/collections.md
Comment (GinaAbrams) [inline src/pages/data-storage/collections.md:21]: One of the goals of the Stacks ecosystem is to give

File: src/pages/data-storage/collections.md
Comment (GinaAbrams) [inline src/pages/data-storage/collections.md:46]: This seems funky, can we remove mentions of "Stacks believes"

File: src/pages/data-storage/overview.md
Comment (GinaAbrams) [inline src/pages/data-storage/overview.md:26]: The Stacks blockchain addresses

File: src/pages/data-storage/overview.md
Comment (GinaAbrams) [inline src/pages/data-storage/overview.md:36]: The Stacks blockchain stores only

File: src/pages/authentication/overview.md
Comment (GinaAbrams) [inline src/pages/authentication/overview.md:123]: Stacks authentication tokens?

**Comment by GinaAbrams** [comment]
Added a couple small comments, looks good otherwise! 🙏

**Comment by friedger** [comment]
@GinaAbrams there is a paragraph about Blockstack PBC and the history. What should be done with it?

Also who issued the stacks tokens?

**Comment by friedger** [comment]
@GinaAbrams  I have addressed your comments.

What about "The Blockstack App" and the "Blockstack Browser"?

The the two section in the overview:

* Blockstack PBC section: https://github.com/blockstack/docs/blob/05ac3f2882773346cfc1a9f371132219d3ce1810/src/pages/ecosystem/overview.md#L16
* Blockstack Token LLC: https://github.com/blockstack/docs/blob/05ac3f2882773346cfc1a9f371132219d3ce1810/src/pages/ecosystem/overview.md#L38

**Comment by GinaAbrams** [comment]
Thanks Friedger! I think we can probably wait on making changes to the PBC overview or Token LLC overview until the Hiro change has been more finalized.

**Review by agraebe — APPROVED** [review]

**Comment by agraebe** [comment]
@GinaAbrams could you help with a final review? I think we could merge this today if all looks good

File: src/pages/data-storage/collections.md
Comment (GinaAbrams) [inline src/pages/data-storage/collections.md:27]: Can we use Stacks here to remain consistent on the language 🙏

**Review by GinaAbrams — COMMENTED** [review]
Added a single small comment. Otherwise lgtm!

File: src/pages/data-storage/collections.md
Comment (friedger) [reply src/pages/data-storage/collections.md]: Updated!

**Comment by agraebe** [comment]
fantastic, thanks for this @friedger. I'll merge once the conflicts are resolved. can you take a look?

**Comment by friedger** [comment]
done

**Comment by agraebe** [comment]
@aulneau could you help here as well? I can't merge this one for some reason

**Comment by agraebe** [comment]
@friedger I can't merge this PR because of a conflict. could you resolve/rebase so we can merge this? thanks!

**Comment by friedger** [comment]
@agraebe done

**Comment by CharlieC3** [comment]
@friedger I think I see what's going on WRT the merge conflict. There's been some commits to `master` since you created this PR, in order to get a smooth merge and fix this, you need to perform a manual _rebase_ in your feature branch. Unfortunately it seems the `master` branch was merged into this feature branch quite a few times. There's a way to fix this properly, but honestly with the amount of merges from `master` performed it'll be simpler to just save the changes you made in this PR and re-create the feature branch from the latest commit in `master`.

For future reference, when you want to pull updates from the `master` branch into your feature branch, use:
`git pull --rebase origin master`

This will lift up the stack of changes you've made in the feature branch, and tuck new master updates underneath them to maintain chronological order, then it places your commits back on top of the commit stack.

You'll then need to _force_ push your branch back up to your fork
`git push <my-fork-name> -f`

I do this everytime I pull in updates from a base branch, and it should help keep PRs clean/mergeable.

**Comment by friedger** [comment]
This PR is replaced by #900

**Pull request closed without merging**
